src/components/ToppingsSelect/index.jsx

```diff
@@ -3,16 +3,41 @@ import Topping from "../Topping";
 import './style.css';
 
 const ToppingsSelect = ({ toppings }) => {
+
+  const [toppingCheck, setToppingCheck] = useState(toppings);
+
+  const handleToppingChange = (index, selected) => {
+    const newToppingCheck = [...toppingCheck];
+    newToppingCheck[index].selected = selected;
+    setToppingCheck(newToppingCheck);
+  };
+
+  let toppingCount = 0;
+  toppingCheck.forEach((topping) => toppingCount += topping.selected);
+
+  let toppingPrice = 0;
+  toppingCheck.forEach((topping) => {
+    let newPrice = topping.price;
+    if ( topping.selected === true) {
+      toppingPrice += newPrice
+    }}
+  )
+
+
```

Comment:
zvládla bys to udělat pomocí filter/reduce? :)) https://stackoverflow.com/questions/46680151/using-reduce-to-get-sum-of-all-numbers-in-an-array-in-javascript

první by se dalo udělat jako:
```
const toppingCount  =   toppingCheck.filter((topping) => topping.selected).length;
```